comment_iowriter.go

```diff
@@ -167,6 +175,8 @@ func (cw *SARIFCommentWriter) Post(_ context.Context, c *Comment) error {
 	return nil
 }
 
+func (*SARIFCommentWriter) ShouldPrependGitRelDir() bool { return false }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method SARIFCommentWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:Cg8xZGExNzIzZjUwN2RhOWMSD2dvbGludC1jaXJjbGVjaQ== -->